api/google-setup.js

```diff
@@ -0,0 +1,31 @@
+const passport = require('passport');
+const GoogleStrategy = require('passport-google-oauth20').Strategy;
+
+
+
+passport.use(new GoogleStrategy({
+    clientID: "798089317931-hca1vqdbgf41bcu3vnbsg4c7f1h58gdu.apps.googleusercontent.com",
+    clientSecret: "jQCNTrDmsy9sTwy8ANjtxCwB",
+    callbackURL: "http://localhost:3000/homepage",
```

Comment:
I think the callbackURL should be /auth/google/redirect because passport.authenticate needs to be called again. Just found out about it too. [here](https://stackoverflow.com/questions/26268271/why-is-passport-authenticate-called-twice) I think this is probably why the setup isn't quite working properly.